src/e2e-test/java/io/cdap/plugin/utils/SalesforceClient.java

```diff
@@ -105,12 +107,12 @@ public static String getAccessToken() {
     return loginAccessToken;
   }
 
-  public static String createLead(JSONObject objectJson, String objectName) throws UnsupportedEncodingException {
+  public static String createObject(JSONObject objectJson, String objectName) throws UnsupportedEncodingException {
     getAccessToken();
     String baseUri = loginInstanceUrl + REST_ENDPOINT + API_VERSION;
     Header oauthHeader = new BasicHeader("Authorization", "Bearer " + loginAccessToken);
     String uri = baseUri + "/sobjects/" + objectName + "/";
-    String leadId = "null";
+    String Id = "null";
```

Comment:
Why Id is String null?

Reply:
we us it here just to initialize default value.